src/OpenGLES/EAGLContext.cs

```diff
@@ -22,7 +22,17 @@ public EAGLContext ()
 		}
 #endif
 
-#if !NET
```

Comment:
Looks like `PresentRenderBuffer (nuint target, double presentationTime)` used to only be available for !NET, should this change be reverted?

Reply:
/cc @rolfbjarne

Reply:
What you did is correct!